routeros/resource_interface_vrrp.go

```diff
@@ -56,7 +56,6 @@ func ResourceInterfaceVrrp() *schema.Resource {
 		"group_master": {
 			Type:        schema.TypeString,
 			Optional:    true,
-			Default:     "none",
```

Comment:
Please check the effect with the "Computed: true," parameter. I don't have a router at hand right now.

Reply:
Right the default value should come from the router API if nothing is provided by the user.
I have tested the change here and it works.